doc/_templates/globaltoc.html

```diff
@@ -5,6 +5,7 @@
   <li><a href="{{ pathto('index') }}">Introduction</a></li>
   <li><a href="{{ pathto('features') }}">Features</a></li>
   <li><a href="{{ pathto('level1') }}">Level 1 spec</a></li>
+  <li><a href="{{ pathto('implementations') }}">Client implementations status</a></li>
```

Comment:
I think i'd just go with "client implementation", not "client implementations".

Reply:
Last commit fix some of your comments. Leyend still needs to be created, versions needs to be added, and we don't have still a way to keep it up to date

Reply:
@Valodim @pbrunschwig @BjarniRunar could you add your MUA logos in the table?.
Maybe we need to create a new column or just replace the name for an icon.
Thanks

Reply:
I'd still recommend s/Client implmentations/Client implementation/ -- that doesn't seem to be resolved yet.